### PR TITLE
Update to follow new template, add one more solution

### PR DESCRIPTION
**Pull request by vaskezu**

### CODE REVIEW

**Comment by Rafase282** [comment]
[New Template](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Challenge-Solution-Template)

**Comment by Rafase282** [comment]
Thanks you, please change the link to a relative one and squash your commits into one. We are working on fixing the relative issues problem.

File: Algorithm-Caesars-Cipher.md [hunk 1]
```diff
@@ -72,21 +129,16 @@ function rot13(str) {
...
+**`thanks @anuragaryan @SaintPeter @vaskezu for your help with Algorithm: Caesar's Cipher`**
+
+## :clipboard: NOTE TO CONTRIBUTORS:
+- :warning: **DO NOT** add solutions that are similar to any existing solutions. If you think it is ***similar but better***, then try to merge (or replace) the existing similar solution.
+- Add an explanation of your solution.
+- Categorize the solution in one of the following categories &mdash; **Basic**, **Intermediate** and **Advanced**. :traffic_light:
+- Please add your username only if you have added any **relevant main contents**. (:warning: ***DO NOT*** *remove any existing usernames*)
 
-> **NOTE:** Please add your username only if you have added any **relevant main contents** to the wiki page. (Please don't remove any existing usernames.)
+> See :point_right: [**`Challenge Solution Template`**](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Challenge-Solution-Template) for reference.
```
Comment (Rafase282) [inline Algorithm-Caesars-Cipher.md]: See :point_right: [**`Challenge Solution Template`**](Challenge-Solution-Template) for reference.

**Comment by vaskezu** [comment]
I think it's good now, this has been a great learning experience. Learn so much about github :+1:

**Comment by Rafase282** [comment]
@vaskezu thanks for the changes, if you can, we could use your help changing other similar challenges to the new template. I'm also glad you are able to learn more about git. This is definitely better than the git lessons we offer. Maybe we should make them do a contribution (optional of course) as a way to help them better understand git.

**Comment by vaskezu** [comment]
Yea I will gladly help with changing other algorithm challenges to the new template too.

**Pull request merged**